packages/design-system/src/components/Button/Button.stories.jsx

```diff
@@ -94,6 +94,297 @@ VariationsOnDark.decorators = [
   ),
 ];
 
+export const AllAnchorButtons = () => {
```

Comment:
Created new story to reflect how the new Button API looks/behaves when there's an `<a>` under the hood. Turned out it was very bad! CSS fixes further down.